src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -355,7 +370,7 @@ type RepliesProps = {
     translations?: Translations,
 };
 
-const Replies = ({
+export const Replies = ({
```

Comment:
Is Replies used anywhere outside of BaseComment? If it isn't, I would advocate for not exporting it (if that wouldn't break anything).

I believe part of the argument for keeping it in this file aside from avoiding a major refractor is that a reply is sort of part of a comment component even from the way it arrives from the back-end. If they are going to be somewhat coupled for right now, I think it would make sense to only allow it to be accessed through the BaseComment.

If we need to export it only for testing, we could add a comment noting that.

Reply:
They were both originally exported. I just moved the export inline so that you know its being exported as soon as you start reading the definition. You are right that Replies is being exported for testing.

About Replies living here, I definitely do not subscribe to the idea that the shape of data from the backend should dictate the composition of ui components. Typically, you'll want to build out leaves in their own files, and then gradually bring them together in increasingly abstract and trunkwise components. It's an awful idea to have needlessly massive components just for the purpose of trying to reflect data structures.

Reply:
Just want to make sure I understand: is the idea that we want to add a comment to warn people off from using `Replies` in any component other than `BaseComment`? If so, it seems a little unnecessary to me. But I also don't mind adding it if you see value in that. Jlmk :)

Reply:
Having the export won't hurt anything. I think it'll be fine to leave.